front/components/vaults/CreateOrEditVaultModal.tsx

```diff
@@ -84,8 +96,43 @@ export function CreateOrEditVaultModal({
     }
   }, [vaultMembers]);
 
-  const router = useRouter();
-  const sendNotification = useContext(SendNotificationsContext);
+  const debouncedFetch = useCallback(
```

Comment:
Let's move this to a hook here: https://github.com/dust-tt/dust/blob/main/front/lib/swr/user.ts